kolibri/plugins/setup_wizard/assets/src/vue/index.vue

```diff
@@ -8,14 +8,19 @@
         <h2 class="title">{{ $tr('deviceOwner') }}</h2>
         <div class="description">{{ $tr('deviceOwnerDescription') }}</div>
         <div class="creation-form">
-          <br><input :class="{ 'input-error': username_error }" type="text" v-model="username" :placeholder="$tr('username')" :aria-label="$tr('username')"><br>
-          <br><input :class="{ 'input-error': password_error }" type="password" v-model="password" :placeholder="$tr('password')" :aria-label="$tr('password')"><br>
-          <br><input :class="{ 'input-error': password_error }" type="password" v-model="confirm_password" :placeholder="$tr('confirmPassword')" :aria-label="$tr('confirmPassword')">
+          <h3>{{ $tr('username') }}:</h3>
+          <input :class="{ 'input-error': username_error }" type="text" v-model="username" :aria-label="$tr('username')">
+          <h3>{{ $tr('password') }}:</h3>
+          <input :class="{ 'input-error': password_error }" type="password" v-model="password" :aria-label="$tr('password')">
+          <h3>{{ $tr('confirmPassword') }}:</h3>
+          <input :class="{ 'input-error': password_error }" type="password" v-model="confirm_password" :aria-label="$tr('confirmPassword')">
```

Comment:
There are ways to polyfill placeholders (with JS) so they show up in older browsers too, but if you're happy with how this looks stylistically, then sounds good.

(We probably have a similar issue in the login modal, where placeholders are used as well)